src/app/common/markdown-editor/markdown-editor.scss

```diff
@@ -126,3 +126,21 @@ $markdown-editor-height: 15px;
     border: 1px solid #ddd;
   }
 }
+
+/////////////////////////////////////////////////////////
+//  <!-- #Code by Maverick - starts --> 
+
+
+// Style for comment type selector icons
```

Comment:
We don't need these tags - they just add to the clutter. Git tracks these changes automatically.